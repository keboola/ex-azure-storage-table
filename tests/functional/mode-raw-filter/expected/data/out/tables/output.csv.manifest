{
    "primary_key": [
        "PartitionKey",
        "RowKey"
    ],
    "columns": [
        "PartitionKey",
        "RowKey",
        "data"
    ],
    "incremental": false
}
