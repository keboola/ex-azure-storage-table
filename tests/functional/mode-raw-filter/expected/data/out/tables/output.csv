"Europe","2","{""PartitionKey"":""Europe"",""RowKey"":""2"",""name"":""Jacob"",""age"":25,""Timestamp"":""***""}"
"Europe","4","{""PartitionKey"":""Europe"",""RowKey"":""4"",""name"":""Michael"",""age"":27,""Timestamp"":""***""}"
